lib/internal/url.js

```diff
@@ -1414,25 +1414,27 @@ const backslashRegEx = /\\/g;
 const newlineRegEx = /\n/g;
 const carriageReturnRegEx = /\r/g;
 const tabRegEx = /\t/g;
+const questionRegex = /\?/g;
+const hashRegex = /#/g;
 
 function encodePathChars(filepath) {
-  if (StringPrototypeIncludes(filepath, '%'))
+  if (StringPrototypeIndexOf(filepath, '%') !== -1)
```

Comment:
What's the reason for this change? It probably deserves a comment.

Reply:
Right now, IndexOf is faster than Includes at the moment, but depending on the V8, any comment I write at the moment can be obsolete.